### ISSUE TITLE
Line indicator for 'scene to inventory' points towards held item rather than desired destination.

### ISSUE THREAD

**Issue by ThijsvbASML** [JvdBosch/vroomissues#34]
When grabbing an item from your inventory to a desired location. Using the help-line (By holding both triggers) does not draw a line while the item is in the inventory, and draws it to the grabbed item after bringing it out of the inventory and floating in the air.

Suggestion to change this behavior so the help-line points to the desired destination of the item, rather than item itself.

**Comment by ThijsvbASML** [comment]
It seems this is more of a keybind problem, it works fine if you use the help-line while you are holding the item already. But seeing the button layout it is impossible to get a help-line while you are holding the item in question.

**Comment by JvdBosch** [comment]
Yes, lines are drawn to items which are grabbable, when you have nothingh grabbed. When you do have something grabbed, it shows a line to the location(s) it can go to. So you mean how to toggle the assist mode, while holding an item?

**Comment by ThijsvbASML** [comment]
In VR this is commonly done by clicking the back triggers of both controllers. However it is not possibly to click the trigger while it is already held down to hold an object. As such we may need to see if we can put it on a different button.

**Comment by JvdBosch** [comment]
Aha, now I understand. Yes, than we might need to re-assign some controller-buttons. Let's talk about that live some time soon.
